curriculum/challenges/english/25-front-end-development/quiz-css-variables/66ed9018f45ce3ece4053eb9.md

```diff
@@ -17,439 +17,445 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+If you’ve declared a custom property for animation speed using `@property`, and a user adjusts their settings for animations in their browser, how would that affect your custom property? Would they be able to see the changes, or does it stick to the value you’ve set?
```

Comment:
I'm not actually certain a user can set that in their browser. Unless I'm missing something, in which case provide a link so I can read up on it. 

https://developer.mozilla.org/en-US/docs/Web/CSS/animation-duration

Reply:
You're right that not all browsers have a direct setting for adjusting animation speed. However, many modern operating systems and browsers support a "reduce motion" option in their accessibility settings. This allows users to minimize or disable animations for a more comfortable experience.

You can also check user preferences with the `prefers-reduced-motion` media query in CSS. If a user has this setting enabled, you can adjust your animations accordingly.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion
https://web.dev/articles/prefers-reduced-motion